js/src/provider/WalletLinkProvider.ts

```diff
@@ -650,45 +651,45 @@ export class WalletLinkProvider implements Web3Provider {
     params: unknown[]
   ): Promise<JSONRPCResponse> {
     this._requireAuthorization()
-    const typedData = params[0]
+    const typedDataJson = ensureJsonOrParseString(params[0])
```

Comment:
call the variable `typedData` since it will always be an `object` rather than a JSON string?

Reply:
c5e0d171e4a6fa75f11ce18f934709047f095034